meshroom/nodes/aliceVision/CameraInit.py

```diff
@@ -123,16 +123,27 @@
         label="Camera Type",
         description="Mathematical model used to represent a camera:\n"
                      " - pinhole: Simplest projective camera model without optical distortion (focal and optical center).\n"
-                     " - radial1: Pinhole camera with one radial distortion parameter.\n"
-                     " - radial3: Pinhole camera with 3 radial distortion parameters.\n"
-                     " - brown: Pinhole camera with 3 radial and 2 tangential distortion parameters.\n"
-                     " - fisheye4: Pinhole camera with 4 distortion parameters suited for fisheye optics (like 120deg FoV).\n"
-                     " - equidistant_r3: Non-projective camera model suited for full-fisheye optics (like 180deg FoV).\n"
+                     " - equidistant: Non-projective camera model suited for full-fisheye optics (180deg FoV).\n",
```

Comment:
```suggestion
                     " - equidistant: Non-projective camera model suited for full-fisheye optics.\n",
```

Reply:
Actually, it's not clear to me what is meant here by non-projective. If it is to say non-central, it is not correct as the 3D rays still meet in a single point (see e.g. figure 2 here https://www.researchgate.net/publication/299374422_Comparison_and_Calibration_of_Mobile_Phone_Fisheye_Lens_and_Regular_Fisheye_Lens_via_Equidistant_Model)

Reply:
I think non-projective means it is not a perspective projection. It is using a spherical projection.

Reply:
a spherical projection like the one for equidistant is still a perspective projection through the center of the sphere.